org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -401,6 +412,11 @@ private synchronized void doNodeInitialisation() {
         }
         logger.debug("{}: Channel initialisation complete", nodeIeeeAddress);
 
+        int pollOrReportingInterval = findMinimumPollOrReportingInterval(channels);
```

Comment:
I wonder if these names can't be changed? How about just ```expectedUpdatePeriod``` or ```refreshPeriod```?

Reply:
Sure, `expectedUpdatePeriod` sounds good.

Reply:
Ta.